docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -364,6 +364,19 @@ This change causes a few breaking changes:
    );
   ```
 
+- The component passed to the `layout` slot no longer receives the `value` prop, instead you can use the `usePickerContext` hook:
+
+  ```diff
+  +import { usePickerContext } from '@mui/x-date-pickers/hooks';
+
+   // This contains a small behavior change.
+   // If the picker receives an invalid date,
+   // the old value would equal `null`.
+   // the new value would equal the invalid date received.
```

Comment:
this feels like something is missing, no?

Reply:
What is?

Reply:
Not sure... this sounds a bit weird. I can't really remember, but I guess it could also be that the diff was wrapped and I was mistaken about it. Reading it now it sounds ok! 🤷